src/util/APIService.js

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { BACKEND_URL } from './consts';
+
+export default class APIService {
+	static url = BACKEND_URL;
+
+	static data = {};
+
+	static Get(endpoint, queries = {}) {
+		if (
+			!(
+				queries &&
+				Object.keys(queries).length === 0 &&
+				queries.constructor === Object
+			)
+		) {
+			return new Promise((resolve, reject) => {
+				const payload = Object.assign(this.data, queries);
+				const esc = encodeURIComponent;
+				const query = Object.keys(payload)
+					.map((k) => `${esc(k)}=${esc(payload[k])}`)
+					.join('&');
+				try {
+					const response = axios
+						.get(`${this.url + endpoint}/?${query}`)
```

Comment:
As I remember you can put params in second argument, so query is maybe redundant https://masteringjs.io/tutorials/axios/get-query-params